index.html

```diff
@@ -1,11 +1,47 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>okej</title>
-</head>
-<body>
-    
-</body>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    
+</head>
+<body>
+
+<div>
+<a href="">mydło |</a>
+<a href="">lodówka |</a>
+<a href="">krzesło |</a>
+</div>
+

```

Comment:
Dodaj, prosze, taka linie pozioma, aby oddzielic poszczegolne menu od siebie i aby latwiej bylo je rozroznic:
```suggestion
<hr>
```